modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/2d-rc-transient-physics.i

```diff
@@ -0,0 +1,97 @@
+# Fluid properties
+mu = 1.1
+rho = 1.1
+cp = 1.1
+k = 1e-3
+
+# Operating conditions
+u_inlet = 1
+T_inlet = 200
+T_solid = 190
+p_outlet = 10
+h_fs = 0.01
+
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 2
+    xmin = 0
+    xmax = 5
+    ymin = -1
+    ymax = 1
+    nx = 50
+    ny = 20
+  []
+[]
+
+[Physics]
+  [NavierStokes]
+    [WCNSFVFlow]
+      [flow]
+        compressibility = 'incompressible'
+
+        density = 'rho'
+        dynamic_viscosity = 'mu'
+
+        initial_velocity = '${u_inlet} 1e-12 0'
```

Comment:
Can we just do 0 instead of 1e-12? In the old days we had to do stuff like this so we didn't divide by a zero velocity norm, but we should be past that

Reply:
done